cypress/support/pages/homeСatalogue.pageObject.js

```diff
@@ -1,21 +1,52 @@
 import PageObject from '../PageObject';
 
+
 class HomeAndCataloguePageObject extends PageObject {
   url = '/index.html';
 
-  clickOnLink(linkName) {
-    cy.contains('.nav-link', linkName)
-      .click();
+  get laptopBtn() {
+    return cy.get("#itemc");
+  }
+  clickLaptopBtn() {
+    this.laptopBtn.click();
+  }
+  get laptopNameBtn() {
+    return cy.contains(".hrefch", "Sony vanio i7");
```

Comment:
There is a spelling mistake in the text 'Sony vanio i7'. It should be corrected to 'Sony vaio i7'.